test/pummel/test-crypto-dh-keys.js

```diff
@@ -21,8 +21,14 @@
 
 'use strict';
 const common = require('../common');
-if (!common.hasCrypto)
+if (!common.hasCrypto) {
   common.skip('node compiled without OpenSSL.');
+}
+
+if ((process.config.variables.arm_version === '6') ||
+  (process.config.variables.arm_version === '7')) {
+  common.skip('Too slow for armv6 and armv7 bots');
+}
```

Comment:
I notice that there's another PR that does this. Maybe it would be worthwhile to add a `common.isSlowArm` type property to `common/index.js` to make this check more reusable?

Reply:
I'm pretty sure the other PR that does this is my pummel-in-CI PR that cherry-picks over this commit.